db/schema.rb

```diff
@@ -276,6 +276,23 @@
     t.boolean "manual", default: false
   end
 
+  create_table "paypal_connections", id: :uuid, default: -> { "uuid_generate_v4()" }, force: :cascade do |t|
+    t.uuid "user_id", null: false
+    t.string "encrypted_refresh_token"
+    t.string "encrypted_refresh_token_iv"
+    t.text "full_name"
+    t.text "email"
+    t.text "country"
+    t.boolean "verified_account"
+    t.text "paypal_account_id"
+    t.boolean "hidden", default: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["user_id"], name: "index_paypal_connections_on_user_id"
+  end
+
+  end
```

Comment:
Seems like this isn't the most up to date schema? Can you make sure this aligns when you drop/recreate your DB like `rails db:drop db:create db:migrate`?